.gitignore

```diff
@@ -0,0 +1 @@
+*.sublime-*
```

Comment:
You never want `*.sublime-*` in any of your projects so that shouldn’t be an issue. The problem is that people will start adding their environment-specific stuff into this project which isn’t desired. I don’t care too much if this gets included but just noted that it’s not ideal.